packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -85,10 +88,11 @@ contract BridgeRouter is Version0, Router {
 
     // ======== Initializer ========
 
-    function initialize(address _tokenRegistry, address _xAppConnectionManager)
+    function initialize(address _tokenRegistry, address _xAppConnectionManager, address _connext)
```

Comment:
initialize can only be called once at the proxy's initial deploy. to add new storage variables after the intial deploy, we need a separate setup function. or else, we need to migrate `initialize` to the version that uses numbered initializations instead of binary y/n